cmd/call-api-endpoint/main_test.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/edulinq/autograder/internal/cmd"
+)
+
+// Use the common main for all tests in this package.
+func TestMain(suite *testing.M) {
+	cmd.CMDServerTestingMain(suite)
+}
+
+// Test to ensure an API endpoint that requires parameters works correctly.
+func TestCallApiEndpointBase(test *testing.T) {
+	testCases := []struct {
+		cmd.CommonCMDTestCase
+		endpoint     string
+		targetEmail  string
+		courseID     string
+		assignmentID string
+		submissionID string
```

Comment:
I really don't like this.
The whole point of the CMD is to handle general endpoints, but here you are limiting test cases to only these params (just like you previously had the CMD CLI arguments).